main.go

```diff
@@ -0,0 +1,22 @@
+//A very simple Go application.
+package main
+
+import (
+	"fmt"
+	"log"
+	"time"
+)
+
+//Greet returns a pleasant greeting
+func Greet(name string) string {
+	return fmt.Sprintf("Hello %s!", name)
+}
+
+func main() {
+	greeting := Greet("change-me")
+	log.Printf(greeting)
+	for {
+		//Don't exit.
```

Comment:
why the need not to exit? (otherwise I get distracted by this red herring)

Reply:
Exiting immediately makes Kubernetes think we crashed, so it restarts, and after a handful of crash-loops it marks the container unhealthy.  I extended the comment to explain.